_providers/daleth.cafe.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Csl. Thomas Lüchow
+status: Teacher and Coach for Adults
+domains: 
+  - daleth.cafe
+server:
+  - type: IMAP
+    socket: TLS
+    hostname: daleth.cafe
+    port: 993
+    username_pattern: EMAIL
+  - type: SMTP
+    socket: STARTTLS
+    hostname: daleth.cafe
+    port: 587
+    username_pattern: EMAIL
+before_login_hint: Wellcome to mehl.cloud
+after_login_hint: Now you are loged in
+strict_tls: true
+last_checked: 
+skip_auto_test: false
+website: https://nein.jetzt
+---
+This ist a chatmail-instance with docker byside. 
```

Comment:
Hm, meinst du es läuft daneben noch docker? Oder läuft chatmail in docker? `byside` ist so kein englisches Wort, vllt kann ich weiterhelfen?

Reply:
**lets use 'parallel'**

Some additional services are usefull, but the most important reason is to load bot as a docker container. 

I'am testing two ways to get some services parallel to chatmail. on Mehl.cloud I edited the nginx.conf, on daleth.cafe I have installed docker on a second IP-Address.